modules/mod_signup/mod_signup.erl

```diff
@@ -266,7 +266,9 @@ send_verify_email(UserId, Ident, Context) ->
         {email, Email},
         {verify_key, Key}
     ],
-    z_email:send_render(Email, "email_verify.tpl", Vars, z_acl:sudo(Context)),
+    {Name, _NameCtx} = z_template:render_to_iolist("_name.tpl", [{id, UserId}], z_acl:sudo(Context)),
+    To = z_email:combine_name_email(z_string:trim(iolist_to_binary(Name)), Email),
```

Comment:
If `Name` will have by some reason unicode `iolist_to_binary` will be crashed.

Reply:
In the 0.x all strings are utf8 byte binaries or lists.  So we can easily serialize.

Reply:
Also the "iolist" in the function name should be "iodata"....  but alas, history.